source/tests/unit/converters_tests.cpp

```diff
@@ -197,7 +197,7 @@ TEST(ConvertersTests, CustomField_ApiErrorAndDescriptionToStatus_IncludesCustomF
 
 TEST(ConvertersTests, QuotedLocalizedDescription_ApiErrorAndDescriptionToStatus_MessageIsEncoded)
 {
-  const auto TEST_MESSAGE = std::string("\"chaîne 😀 localisée\"");
+  const auto TEST_MESSAGE = std::string(u8"\"chaîne 😀 localisée\"");
```

Comment:
I'm going to revert this change from this PR, which is for RF error strings. I generally avoid this trap and every once in awhile I'm reminded why including something extra "just because" is a bad idea. 

@bkeryan  I've added links to your comments here into the bug about encoding. It feels related enough to address when specifically fixing and testing that.